requirements.txt

```diff
@@ -29,7 +29,7 @@ python-dateutil==2.8.2                                      # Extensions to the
 stomp.py==6.1.1                                             # ActiveMQ Messaging Protocol
 statsd==3.3.0                                               # Needed to log into graphite with more than 1 Hz
 geoip2==4.5.0                                               # GeoIP2 API (for IPv6 support)
-oauth2client==4.1.3                                         # OAuth 2.0 client library
+google-auth==2.6.0                                          # Google authentication library for Python
```

Comment:
You need to add this too to the `server_requirements_table` in https://github.com/rucio/rucio/blob/master/setuputil.py